### PR TITLE
Create ui-c8y-1018-503-124-info-gauge-widget-should-support-device-type-dashboards-7312.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by reey

Ticket: [MTM-61069](https://cumulocity.atlassian.net/browse/MTM-61069)
Original PR: [7312](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7312)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-124-info-gauge-widget-should-support-device-type-dashboards-7312.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: info gauge widget should support device type dashboards (#7312)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-124-info-gauge-widget-should-support-device-type-dashboards-7312.md]: The Claude AI suggested to use the following title:
```suggestion
title: Info gauge widget now supports device type dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-124-info-gauge-widget-should-support-device-type-dashboards-7312.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61069
+version: 1018.503.124
+---
+info gauge widget should support device type dashboards (#7312)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-124-info-gauge-widget-should-support-device-type-dashboards-7312.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, the info gauge widget did not support device type dashboards, limiting its usability for users who rely on these dashboards. With this change, the info gauge widget has been updated to fully support device type dashboards. This enhancement allows users to effectively utilize the info gauge widget within their device type dashboard setups, providing them with a more comprehensive and flexible monitoring experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@reey I copied the text from the CD change log. Please approve.

**Review by reey — APPROVED** [review]

**Pull request merged**